packages/web/tests/components/NavBar/NavProfileMenu.test.tsx

```diff
@@ -1,15 +1,44 @@
 import React from 'react';
-import { render, screen } from '../../testUtils/testTools';
+import { render, screen, act } from '../../testUtils/testTools';
 import { NavProfileMenu } from '../../../src/components/NavBar/NavProfileMenu';
+import fetchMock from 'fetch-mock-jest';
+
+const mockCurrentUser = {
+  id: 123,
+};
+
+const wait = () => new Promise<void>((resolve) => setTimeout(() => resolve(), 0));
 
 describe('NavLink Components', () => {
-  it('renders correctly', async () => {
+  beforeEach(async () => {
+    jest.clearAllMocks();
+    fetchMock.reset();
```

Comment:
We don't need this because we already have it in [our test setup file](https://github.com/AmericanAirlines/Project-X/blob/213971c6cc6574db11fd60d15f5e937dfd06fd2c/packages/web/tests/setupTests.ts#L3)
```suggestion
```